cumulus_library/base_table_builder.py

```diff
@@ -38,13 +38,13 @@ def execute_queries(self, cursor: object, schema: str, verbose: bool):
         :param schema: A schema name
         :param verbose: toggle for verbose output mode
         """
+        self.prepare_queries(cursor, schema)
```

Comment:
this was moved due to needing to create an extra progress bar inside a builder, since only one progress object can be active at a time (though it can have multiple tasks).